### ISSUE TITLE
Can't watch anime

### ISSUE THREAD

**Issue by lompidesu** [jmir1/aniyomi#263]
### Steps to reproduce

![Screenshot_20211128_131301](https://user-images.githubusercontent.com/95158489/143731191-71805ff9-1518-480d-88ce-4197d4689664.jpg)

I don't know how to fix this. 

### Expected behavior

Fixed. 

### Actual behavior

It suddenly happens, it wont play any anime

### Aniyomi version

0.12.3.5

### Android version

Android 8

### Device

Vivo y18i

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3.5](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by jmir1** [comment]
This is not an issue with the app, the domain dood.ws is being blocked by your ISP. Try using a vpn service or a different DNS server.
